### ISSUE TITLE
Image can't start

### ISSUE THREAD

**Issue by wopox1337** [startersclan/docker-hlstatsxce-daemon#29]
```bash
root@d-homeserver: docker run -it \
    -p 27500:27500/udp \
    -e LOG_LEVEL=1 \
    -e MODE=Normal \
    -e DB_HOST=db \
    -e DB_NAME=hlstatsxce \
    -e DB_USER=hlstatsxce \
    -e DB_PASSWORD=hlstatsxce \
    startersclan/docker-hlstatsxce-daemon:v1.6.19-geoip-alpine-3.8
2023-10-24 02:42:27 Starting daemon...

Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/^weapon { <-- HERE  steam_id: 'STEAM_\d+:(.+?)', weapon_id: (\d+), class: \d+, team: \d+, shots: \((\d+),(\d+)\), hits: \((\d+),(\d+)\), damage: \((\d+),(\d+)\), headshots: \((\d+),(\d+)\), kills: \(\d+,\d+\) }$/ at ./hlstats.pl line 3329.
```

`:latest` also checked

**Comment by leojonathanoh** [comment]
Yes, it cannot, because the original code was written for Perl 4, and the image contains Perl 5. I've made it clear in the readme under the [FAQ](https://github.com/startersclan/docker-hlstatsxce-daemon#q-perl-errors-on-startup). The only way it would be able to start is if you had a custom image with fixes for the bugs.

**Comment by leojonathanoh** [comment]
Perhaps, if you know how, you could use the perl `/scripts` from https://github.com/NomisCZ/hlstatsx-community-edition, or https://github.com/A1mDev/hlstatsx-community-edition, which probably contains the fixes for these bugs, but which may also contain certain more than necessary changes which I haven't tested myself. 

Let me know if help chime in if you need help with how to build a custom image, based on this image, but with the perl `/scripts` of either of those repositories 😄

**Comment by leojonathanoh** [comment]
Hey @wopox1337, just wanted to share that some of my work to dockerize the entire HLStatsX:CE stack in [this fork](https://github.com/startersclan/hlstatsx-community-edition) based on https://github.com/A1mDev/hlstatsx-community-edition. The daemon image works nicely with all the bug fixes needed, and seems to be working from my testing. 

```sh
docker run --rm -it -p 27500:27500/udp startersclan/hlstatsx-community-edition:1.7.0-daemon --help
```

Hope it works out for you 😃

**Comment by wopox1337** [comment]
Thank you @leojonathanoh ! I will check it out soon, and will report back to you with the results.

**Comment by leojonathanoh** [comment]
Glad to help 😄 There's still some bugs to touch up there, so there may be a few fixes to add over the next few days for v1.7.x :)
